examples/createtool/createtool.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Cindy JS</title>
+        <style>
+            body {
+                margin: 0;
+                padding: 0;
+            }
+
+            button {
+                background-color: #FFFFFF;
+                border: 1px solid #CCCCCC;
+            }
```

Comment:
Thanks for the hint. But I'd like to see SVG buttons for our tools soon, in which case we won't be needing PNG backgrounds any longer.